llvm_mode/Makefile

```diff
@@ -335,12 +335,7 @@ endif
 	if [ -f ../split-switches-pass.so ]; then set -e; install -m 755 ../split-switches-pass.so $${DESTDIR}$(HELPER_PATH); fi
 	if [ -f ../cmplog-instructions-pass.so ]; then set -e; install -m 755 ../cmplog-*-pass.so $${DESTDIR}$(HELPER_PATH); fi
 	set -e; if [ -f ../afl-clang-fast ] ; then ln -sf ../afl-clang-fast $${DESTDIR}$(BIN_PATH)/afl-clang ; ln -sf ../afl-clang-fast $${DESTDIR}$(BIN_PATH)/afl-clang++ ; else ln -sf ../afl-gcc $${DESTDIR}$(BIN_PATH)/afl-clang ; ln -sf ../afl-gcc $${DESTDIR}$(BIN_PATH)/afl-clang++; fi
-	install -m 644 -T README.laf-intel.md $${DESTDIR}$(DOC_PATH)/README.llvm_mode.laf-intel.md
```

Comment:
No, I moved them from the master Makefile to here!!! Now they are missing.

Reply:
as previously announced in the chat, I would like to handle installation of the docs in the local Makefiles instead of the toplevel Makefile. Makes more sense IMO. Target names reflect the structure and are collision free then.

Reply:
it is still there just better:
```
        install -m 644 README.*.md $${DESTDIR}$(DOC_PATH)/
```

Reply:
and README.md is still installed as README.llvm_mode.md below